src/script/cpatch.py

```diff
@@ -456,8 +467,10 @@ def build(self):
     def _container_build(self):
         log.info("Building container image")
         cmd = [self._ctx.engine, "build", "--tag", self._ctx.target, "."]
+        cmd.append('--net=host')
```

Comment:
Is this actually part of the new feature, or did we just generally want `--net=host` as well?

Reply:
it's not directly related to the previous change but as noted in the commit message - I do tricky things on my host sometimes and it was eaiser to get everything working with host networking. I didn't feel like pushing a 2nd pr just for this.